infernyx/rule_helpers.py

```diff
@@ -736,3 +736,59 @@ def clean_assa_impression(parts, params):
         yield parts
     except Exception:
         pass
+
+
+def as_user_counting_map(parts, params):
```

Comment:
You can get rid of this function.  In the `activity_stream_user_counting` rule, just set:
    key_parts=['date', 'release_channel', 'version', 'country_code'],
    value_parts=['client_id'],

and continue with your overriding of the combiner and reducer